### PR TITLE
Menu mobile

### PR DESCRIPTION
**Pull request by The-Normal-One1**
> Edit the Html file for the nav bar.
> Edit the CSS file that is suitable for the hamburger button
> Add the javascript file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,24 +24,23 @@
...
-          onclick="myFunction()"
-        >
-          <i class="fa fa-bars"></i>
-        </a>
+        <a class="logo" href="#home">Ermiyas Demsew</a>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
```
Comment (Bondok6) [inline index.html:32]: - [x] According to the requirements, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). so I suggest adding some space (padding-right or margin-right) to the hamburger button, and center the hamburger button and the logo vertically to match your Figma design 👍

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

File: style.css [hunk 2]
```diff
@@ -39,13 +41,85 @@
...
+    height: 100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: #3c3a39;
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
+    box-shadow: 0 10px 27px rgba(0, 0, 0, 0.05);
+  }
```
Comment (Bondok6) [inline style.css]: - [x] According to the requirements, The mobile menu should  follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) (e.g., font, colors, images, text, margins), so the logo should disappear when you open the menu, and kindly decrease the space between the menu-items to match your Figma design 👍💯

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi Team 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Great job implementing the functionality of the mobile menu using JS ✅
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅
- Professional README ✅

## Required Changes ♻️
- [x] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (The-Normal-One1) [reply index.html:32]: I improved my work.

File: style.css (hunk 2 above)
Comment (The-Normal-One1) [reply style.css]: I corrected the spacing

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:32]: Implemented changes 👍

File: style.css [hunk 3]
```diff
@@ -39,15 +42,99 @@
...
+
+  .material-icons {
+    text-decoration: none;
+    padding-bottom: 26px;
+    padding-left: 15px;
+    color: #fff;
+    height: 4px;
+    width: 12px;
+    font-size: 25px;
+    margin-left: 30px;
```
Comment (sja-thedude) [inline style.css]: - [x] Great job so far, Just one small thing is left which is important while following the Figma design.
As you can see below video you have to follow the same colour for menu item options (i.e. #FFF5E1 ) and for the close (X) button (i.e. #67798E).

https://user-images.githubusercontent.com/70269507/195112183-f5867580-db37-4d67-a47e-22a0b1539610.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @The-Normal-One1, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [x] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by The-Normal-One1** [comment]
@sja-thedude 
I corrected the required changes

Can you please check my changes

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @The-Normal-One1,

Good job on making all the changes asked by the previous reviewers.
Your project is complete! 

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ Good descriptive PR 👍
✅ Mobile menu works fine as expected 👍
✅ Nice animation implemented on the mobile menu open and close 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**